docs/earthfile/builtin-args.md

```diff
@@ -36,6 +36,8 @@ The following builtin args are available
 | `EARTHLY_GIT_ORIGIN_URL` | The git URL detected within the build context directory. If no git directory is detected, then the value is an empty string. Please note that this may be inconsistent, depending on whether an HTTPS or SSH URL was used. | `git@github.com:bar/buz.git` or `https://github.com/bar/buz.git` |
 | `EARTHLY_GIT_PROJECT_NAME` | The git project name from within the git URL detected within the build context directory. If no git directory is detected, then the value is an empty string. | `bar/buz` |
 | `EARTHLY_GIT_COMMIT_TIMESTAMP` | The timestamp, as unix seconds, of the git commit detected within the build context directory. If no git directory is detected, then the value is an empty string. | `1626881847` |
+| `EARTHLY_VERSION` | The version of Earthly currently running. | `v0.6.2`, `dev-main` |
```

Comment:
Since this new feature won't be available until it is released, we have to be careful about when we update docs.

Any new docs for unreleased code needs to be submitted in a seperate PR against the `next` branch: https://github.com/earthly/earthly/tree/next

Then when we perform the release we merge next into main, which causes the docs to get updated.

Sorry I forgot to mention this when we were pairing on the issue.

We might also want to change the example to only list `v0.6.2` and omit `dev-main`, since most users won't see that.

Reply:
Makes sense!

Reply:
Moved to new branch: https://github.com/earthly/earthly/tree/brandon/docs-builtin-args
Will PR that docs branch once we're settled on the changes here